src/renderer.ts

```diff
@@ -50,6 +50,32 @@ const Anchor = (props: { href: string; text: string; pillar: Pillar }): ReactEle
         props.text,
     );
 
+const listStyles = (colour: string): SerializedStyles => css`
+    list-style: none;
+    padding-left: 0;
+`
+
+const listItemStyles: SerializedStyles = css`
+    padding-left: 2rem;
+    line-height: 2.2rem;
+    padding-bottom: 0.375rem;
+
+    &::before {
+        display: inline-block;
+        content: '';
+        border-radius: 0.5rem;
+        height: 1rem;
+        width: 1rem;
+        margin-right: 1rem;
+        background-color: ${palette.neutral[86]};
+        margin-left: -2rem;
+    }
```

Comment:
I think if you use `list-style-position: inside` in `listStyles` then you will be able to drop the `padding-left: 2rem` and `margin-left: -2rem` in `listItemStyles`.

Reply:
Ah that looked like an improvement but it's not working with out custom bullet points made from a pseudo ::Before